src/components/modals/TransferPDNTModal/TransferPDNTModal.tsx

```diff
@@ -147,12 +171,12 @@ function TransferPDNTModal({
               maxLength={43}
               value={toAddress}
               setValue={setToAddress}
-              validityCallback={(validity: boolean) =>
-                setIsValidAddress(validity)
-              }
+              validityCallback={(validity: boolean) => validity}
               validationPredicates={{
                 [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: (id: string) =>
                   arweaveDataProvider.validateArweaveId(id),
+                [VALIDATION_INPUT_TYPES.ARWEAVE_ADDRESS]: (id: string) =>
+                  arweaveDataProvider.validateArweaveAddress(id),
```

Comment:
e.g. change the shape of this to be
```suggestion
                [VALIDATION_INPUT_TYPES.ARWEAVE_ADDRESS]: {
                required: false,
                message: 'This address does not look like an Arweave Address.'
                fn: (id: string) => arweaveDataProvider.validateArweaveAddress(id),
```
and within the ValidationInput display the `message` and update `validity` based on if the validation is `required`

Reply:
added